R/coerce.R

```diff
@@ -7,22 +7,7 @@ coerce <- function(x, type) {
 coerce_lgl <- function(x) coerce(x, "logical")
 coerce_int <- function(x) coerce(x, "integer")
 coerce_dbl <- function(x) coerce(x, "double")
-coerce_chr <- function(x) {
-  local_deprecation_user_env()
-  coerce(x, "character")
-}
-
-
-deprecate_to_char <- function(type) {
-  lifecycle::deprecate_warn(
-    "1.0.0",
-    I(paste0("Automatic coercion from ", type, " to character")),
-    I("an explicit call to `as.character()` within `map_chr()`"),
-    always = TRUE,
-    user_env = the$deprecation_user_env
-  )
-}
-
+coerce_chr <- function(x) coerce(x, "character")
```

Comment:
I think it would be worthwile using `depecrate_stop()` here as the error message is more friendly and provides a suggested fix. The warnings might have flown under the radar.

Reply:
I'll take a look at the revdep failures and see if it's causing much pain.

Reply:
There were only four failures (out of ~2000 packages) so I think it's just as easy to patch those packages.